examples/01-basic/06-keyboard-shortcuts/README.md

```diff
@@ -1,9 +1,11 @@
 # Keyboard Shortcuts
 
+TODO: useful example? Non-basic?
```

Comment:
I would keep it, since in the docs ig we can just say that `BlockNoteView` takes the same props as a `div` would. I think the example is useful to show people that this is indeed the preferred method of setting things like keyboard shortcuts.